frontend/src/App.css

```diff
@@ -30,6 +30,10 @@ body {
   background-color: #f8f8f8;
 }
 
+.text-ink {
+  color: #21272D
```

Comment:
IDK if y'all have discussed this, but if you converted your CSS to Sass, you could [override the USWDS theme token colors](https://designsystem.digital.gov/design-tokens/color/theme-tokens/#customizing-theme-color-tokens) (and do a lot of other things that could help manage styles).